tests/e2e/contracts/multi_read_erc20.py

```
@@ -0,0 +1,28 @@
+class multi_read_erc20(gl.Contract):
+    balances: TreeMap[Address, TreeMap[Address, u256]]
+
+    def __init__(self):
+        pass
+
+    @gl.public.write
+    def update_token_balances(
+        self, account_address: str, token_contracts: list[str]
+    ) -> None:
+        for token_contract in token_contracts:
+            contract = gl.ContractAt(Address(token_contract))
```

Ruff (0.11.9)

`gl` may be undefined, or defined from star imports

(F405)